packages/@aws-cdk/aws-appmesh/lib/index.ts

```diff
@@ -17,3 +17,5 @@ export * from './gateway-route';
 export * from './gateway-route-spec';
 export * from './client-policy';
 export * from './health-checks';
+export { TlsListener } from './tls-listener';
+export { TlsMode } from './tls-listener';
```

Comment:
Can you replace this with

```suggestion
export * from './tls-listener';
```

Reply:
Addressed.